llvm/lib/Transforms/Instrumentation/MemorySanitizer.cpp

```diff
@@ -899,8 +899,8 @@ void MemorySanitizer::createUserspaceApi(Module &M,
       getOrInsertGlobal(M, "__msan_va_arg_origin_tls",
                         ArrayType::get(OriginTy, kParamTLSSize / 4));
 
-  VAArgOverflowSizeTLS =
-      getOrInsertGlobal(M, "__msan_va_arg_overflow_size_tls", IRB.getInt64Ty());
+  VAArgOverflowSizeTLS = getOrInsertGlobal(M, "__msan_va_arg_overflow_size_tls",
```

Comment:
this one is defined at llvm-project/compiler-rt/lib/msan/msan.cpp
we need it to change is this patch as well